segfast/memmap_loader.py

```diff
@@ -403,7 +418,7 @@ def read_chunk(path, shape, offset, dtype, headers, start, chunk_size):
 
     buffer = np.empty((chunk_size, len(headers)), dtype=np.int32)
     for i, header in enumerate(headers):
-        buffer[:, i] = mmap[header][start : start + chunk_size]
+        buffer[:, i] = mmap[header.name][start : start + chunk_size]
     return buffer
```

Comment:
Looks like the whole function can be simplified considering the comment above:
```suggestion
    mmap = np.memmap(filename=path, mode='r', shape=shape, offset=offset, dtype=dtype)
    return np.array(mmap[[header.name for header in headers]][start : start + chunk_size])
```

`array` is used here to convert `mmap` to an instace of `array`